tests/E2E Tests/WebAppUiTests/UiTestHelpers.cs

```diff
@@ -309,6 +310,64 @@ internal static async Task<string> GetValueFromKeyvaultWitDefaultCreds(Uri keyva
             SecretClient client = new(keyvaultUri, creds);
             return (await client.GetSecretAsync(keyvaultSecretName)).Value.Value;
         }
+
+        internal static bool StartAndVerifyProcessesAreRunning(List<ProcessStartOptions> processDataEntries, out Dictionary<string, Process> processes)
+        {
+            processes = new Dictionary<string, Process>();
+
+            //Start Processes
+            foreach (ProcessStartOptions processDataEntry in processDataEntries)
+            {
+                var process = UiTestHelpers.StartProcessLocally(
+                                                processDataEntry.TestAssemblyLocation,
+                                                processDataEntry.AppLocation,
+                                                processDataEntry.ExecutableName,
+                                                processDataEntry.EnvironmentVariables);
+
+                processes.Add(processDataEntry.ExecutableName, process);
+                Thread.Sleep(5000);

```

Comment:
nit: await Task.Delay(5000)

Reply:
I am trying to avoid making these methods async because they prevent the use of the out parameter. I guess I can return the result as a Tuple